test/index.md

```diff
@@ -95,3 +99,31 @@ This is a Ace field
 ## Fields inside iframe
 
 <iframe src="." frameborder="0" class="field">One level deep, please!</iframe>
+
+## Monaco
+
+<script src="https://unpkg.com/monaco-editor@latest/min/vs/loader.js"></script>
+<div id="monaco-editor"></div>
+
+<script>
+	require.config({ paths: { 'vs': 'https://unpkg.com/monaco-editor@latest/min/vs' }});
+	window.MonacoEnvironment = { getWorkerUrl: () => proxy };
+
+	let proxy = URL.createObjectURL(new Blob([`
+		self.MonacoEnvironment = {
+			baseUrl: 'https://unpkg.com/monaco-editor@latest/min/'
+		};
+		importScripts('https://unpkg.com/monaco-editor@latest/min/vs/base/worker/workerMain.js');
+	`], { type: 'text/javascript' }));
+
+	require(["vs/editor/editor.main"], function () {
+		let editor = monaco.editor.create(document.getElementById('monaco-editor'), {
+			value: [
+				'function x() {',
+				'\tconsole.log("Hello world!");',
+				'}'
+			].join('\n'),
+			language: 'javascript',
+		});
+	});
```

Comment:
This looks like a lot of code just initialize a field. Refer to the docs, it's a simple call

https://github.com/microsoft/monaco-editor/blob/main/docs/integrate-esm.md

You can also use `<script type=module>import Monaco from "https://CDN url here"` to load the file from the CDN

Reply:
Ugh it looks like that's not supported: 

- https://github.com/microsoft/monaco-editor/issues/2335